gender_novels/corpus_gen.py

```diff
@@ -405,7 +454,7 @@ def get_country_publication_wikidata(author, title):
     # try to match country_id to major English-speaking countries
     if (country_id == "Q30"):
```

Comment:
make a dictionary elsewhere:

```
country_id_to_name = {
   'Q30': 'United States',
   'Q145': 'United Kingdom',
   ...
   'Q288622': 'Narnia',
}

# then

if country_id in country_id_to_name:
    return country_id_to_name[country_id]
```

then it becomes an easy data task for a new programmer (or an automatic scrapping script) to add other countries in there later.